nsis/SharedShortcuts.nsh

```diff
@@ -1,13 +1,13 @@
 Section "Start Menu Shortcuts" SEC_STARTMENU
 
   ; CreateShortCut gets the working directory from OutPath
-  SetOutPath "$INSTDIR"
+  SetOutPath "$INSTDIR\PCSX2 ${APP_VERSION}"
```

Comment:
RedDevilus has pointed out on Discord that empty PCSX2 1.6.0 folders are created within the folders.

It's because of the changes to this file. Revert them and it should be fixed.